### DISCUSSION TITLE
Why do we need a trailing slash after setting path_prefix for management plugin ?

### DISCUSSION THREAD

**Discussion by claireleray-poclain** [rabbitmq/rabbitmq-server#7309] [Questions]
Hi!

The [documentation](https://www.rabbitmq.com/management.html#path-prefix) specifies that I can set `management.path_prefix` to use a prefix such as /my-prefix for HTTP requests to the management plugin. However, it also says that the URL to access the plugin will be `host:port/my-prefix/`, with the trailing slash.

This behavior works great, but I find it pretty confusing. What is the reason ? Is there a workaround to redirect `host:port/my-prefix` to `host:port/my-prefix/` ?

**Comment by michaelklishin** [comment]
The plugin does not allow you to set up a redirect but you can put in a reverse proxy such as Nginx in front of the API (a good idea for many reasons for production anyway) and set up any URI rewrites you please.
